research/README.md

```diff
@@ -0,0 +1,29 @@
+# Research
```

Comment:
This is just a directory listing, there is nothing to talk about...

Reply:
Yep, makes sense. Please make sure to add a docstring to `2d-data.py` so that it's clear what that file does, no lessons to really note since it's just integration. :)